### ISSUE TITLE
Crash during document capture

### ISSUE THREAD

**Issue by clapslock** [onfido/onfido-ios-sdk#191]
### What was the expected behaviour?

App did not crash. 

### What happened instead?

App crashed. 

### Version info:

- Onfido SDK (Debug/Release): `18.3.1`
- iOS: `13.4.1`
- Xcode: `11.7`
- Device/Simulator: Device; iPhone 7 Plus

### Stack trace

```
Fatal Exception: NSInvalidArgumentException
0  CoreFoundation                 0x1bc2cd5f0 __exceptionPreprocess
1  libobjc.A.dylib                0x1bbfefbcc objc_exception_throw
2  AVFoundation                   0x1c6605d1c -[AVCapturePhotoOutput capturePhotoWithSettings:delegate:]
3  Onfido                         0x104188130 CameraLayerImpl.capturePhoto(onCompletion:) + 133 (__hidden#5540_:133)
4  Onfido                         0x104267ce0 PhotoCaptureView.didTapCaptureButton() + 282 (__hidden#17258_:282)
5  Onfido                         0x104262a6c _hidden#17065_ (__hidden#1867_)
6  UIKitCore                      0x1c0379574 -[UIApplication sendAction:to:from:forEvent:]
7  UIKitCore                      0x1bfd8c278 -[UIControl sendAction:to:forEvent:]
8  UIKitCore                      0x1bfd8c5d0 -[UIControl _sendActionsForEvents:withEvent:]
9  UIKitCore                      0x1bfd8b5f4 -[UIControl touchesEnded:withEvent:]
10 UIKitCore                      0x1c03b3250 -[UIWindow _sendTouchesForEvent:]
11 UIKitCore                      0x1c03b4a18 -[UIWindow sendEvent:]
12 UIKitCore                      0x1c03906a0 -[UIApplication sendEvent:]
13 UIKitCore                      0x1c0410d84 __dispatchPreprocessedEventFromEventQueue
14 UIKitCore                      0x1c04138bc __handleEventQueueInternal
15 UIKitCore                      0x1c040b8d0 __handleHIDEventFetcherDrain
16 CoreFoundation                 0x1bc24bb64 __CFRUNLOOP_IS_CALLING_OUT_TO_A_SOURCE0_PERFORM_FUNCTION__
17 CoreFoundation                 0x1bc24babc __CFRunLoopDoSource0
18 CoreFoundation                 0x1bc24b244 __CFRunLoopDoSources0
19 CoreFoundation                 0x1bc246274 __CFRunLoopRun
20 CoreFoundation                 0x1bc245c34 CFRunLoopRunSpecific
21 GraphicsServices               0x1c638f38c GSEventRunModal
22 UIKitCore                      0x1c037822c UIApplicationMain
23 Azimo                          0x100edb428 main + 15 (main.m:15)
24 libdyld.dylib                  0x1bc0cd800 start
```

**Comment by blunt008** [comment]
Hi, 

Can you please let us know the following:
- Does this happen every time and if not how often?
- Can you share with us your SDK configuration?

Many thanks,
Damian

**Comment by clapslock** [comment]
Hi there, 

(1) This was a single crash 
(2) Sure: https://gist.github.com/clapslock/25839e8c32e338e33e0888eeb985d8f6

We plan to update the SDK to 18.8.0 in a week or so this might not even be relevant but I thought I'd share it just for completeness sake.

**Comment by blunt008** [comment]
Hi Bartosz,

Sorry for the wait! We've managed to fix several issues in our latest release 18.8.1. We strongly recommend this upgrade if you were facing any sort of errors during document capture process. 

Many thanks,
Damian

**Comment by keremgund** [comment]
Closing the issue due to lack of response. 
Please open new one if you need further assistance.

Thanks,
Kerem
